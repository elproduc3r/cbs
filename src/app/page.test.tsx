import React from 'react';
import { render, screen } from '@testing-library/react';

import Page from './page';

describe('<Page />', () => {
  test('renders button', () => {
    render(<Page />);
    const button = screen.getByRole("button");
    const message = screen.getByText(/watch live/i);
    const message2 = screen.getByText(/cbs sports/i);
    expect(message).toBeInTheDocument();
    expect(message2).toBeInTheDocument();
    expect(button).toBeInTheDocument();
  });  
});
