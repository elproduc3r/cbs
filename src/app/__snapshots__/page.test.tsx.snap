// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> renders button 1`] = `
<div>
  <div
    class="background-container"
  >
    <div>
      Context: 
      1
    </div>
    <div
      class="message-container"
    >
      Watch live
      <br />
      CBS Sports
      <button
        class="start-button"
      >
        Get Started
      </button>
    </div>
  </div>
  <div
    class="background-gradient"
  />
</div>
`;
